test_rmw_implementation/test/test_serialize_deserialize.cpp

```diff
@@ -112,6 +119,67 @@ TEST_F(CLASSNAME(TestSerializeDeserialize, RMW_IMPLEMENTATION), clean_round_trip
     rmw_get_error_string().str;
 }
 
+TEST_F(CLASSNAME(TestSerializeDeserialize, RMW_IMPLEMENTATION), clean_round_trip_for_c_bounded_message) {
+  const rosidl_message_type_support_t * ts{
+    ROSIDL_GET_MSG_TYPE_SUPPORT(test_msgs, msg, BoundedSequencesNoStrings)};
```

Comment:
@MiguelCompany hmm, did you mean
```suggestion
    ROSIDL_GET_MSG_TYPE_SUPPORT(test_msgs, msg, BoundedPlainSequences)};
```
? Same elsewhere.

Reply:
Ups! Fixed on f776a16f9bd5c6924621195b88373e2e4dfa0a96